graphql/schema/types/patch.graphql

```diff
@@ -74,6 +74,7 @@ type Patch {
   project: PatchProject
   projectID: String!
   projectIdentifier: String!
+  projectMetadata: Project
```

Comment:
I wonder if it's necessary to send the entire `Project`, if we only really need the `owner` and `repo`. It also kind of makes the `projectIdentifier` patch resolver redundant. Would it make sense to instead send an object with those two fields?